apps/condo/domains/marketplace/tasks/cancelOldInvoices.spec.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @jest-environment node
+ */
+
+const index = require('@app/condo/index')
+const dayjs = require('dayjs')
+
+const { makeLoggedInAdminClient, setFakeClientMode } = require('@open-condo/keystone/test.utils')
+
+const { createTestAcquiringIntegration } = require('@condo/domains/acquiring/utils/testSchema')
+const { createTestBillingIntegration } = require('@condo/domains/billing/utils/testSchema')
+const {
+    INVOICE_CONTEXT_STATUS_FINISHED,
+    DAYS_TO_CANCEL_PUBLISHED_INVOICES, INVOICE_STATUS_CANCELED, INVOICE_STATUS_PUBLISHED,
+} = require('@condo/domains/marketplace/constants')
+const { cancelOldInvoices } = require('@condo/domains/marketplace/tasks/cancelOldInvoices')
+const {
+    createTestInvoiceContext,
+    createTestInvoice,
+    updateTestInvoice, Invoice,
+} = require('@condo/domains/marketplace/utils/testSchema')
+const { createTestOrganization } = require('@condo/domains/organization/utils/testSchema')
+
+let adminClient
+let dummyO10n, dummyIntegration, dummyInvoiceContext
```

Comment:
Why not just `organization, integration, invoiceContext`?

Reply:
To keep these names for use within particular tests if needed :)